lang/python27/Portfile

```diff
@@ -61,43 +63,48 @@ depends_lib         port:bzip2 \
                     port:ncurses \
                     port:sqlite3 \
                     port:zlib
-depends_run         port:python_select \
+    depends_run     port:python_select \
                     port:python2_select
 
-openssl.branch      1.1
+
+    openssl.branch  1.1
+}
 
 # This port is used by clang-3.4 to bootstrap libcxx
 subport ${name}-bootstrap {
-    set stdprefix ${prefix}
-    prefix ${prefix}/libexec/libcxx-bootstrap
+    revision                [ expr ${revision} + 1 ]
```

Comment:
Remember to [brace your expressions](https://wiki.tcl-lang.org/page/Brace+your+expr%2Dessions). :)